Formula/jetty.rb

```diff
@@ -31,6 +43,22 @@ def install
     Dir.glob(libexec/"bin/*.sh") do |f|
       (bin/File.basename(f, ".sh")).write_env_script f, env
     end
+    return if Hardware::CPU.intel?
+
+    # We build a native ARM libsetuid to enable Jetty's SetUID feature.
+    # https://www.eclipse.org/jetty/documentation/jetty-9/index.html#configuring-jetty-setuid-feature
+    libexec.glob("lib/setuid/libsetuid-*.so").map(&:unlink)
+    resource("jetty.toolchain").stage do
+      cd "jetty-setuid"
+      # Fix header paths on macOS as they are set to old system Java
+      inreplace "libsetuid-osx/pom.xml" do |s|
+        s.gsub! "<javahOS>mac<", "<javahOS>darwin<"
+        s.gsub!(/(<jdkIncludePath>)[^<]*</, "\\1#{Language::Java.java_home}/include<")
+      end
```

Comment:
Can we upstream this?

Reply:
I can try opening a PR for following diff, but not sure if upstream is still building with the existing paths.
```diff
diff --git a/jetty-setuid/libsetuid-osx/pom.xml b/jetty-setuid/libsetuid-osx/pom.xml
index a91a311..3ea89db 100644
--- a/jetty-setuid/libsetuid-osx/pom.xml
+++ b/jetty-setuid/libsetuid-osx/pom.xml
@@ -44,8 +44,8 @@
         <artifactId>native-maven-plugin</artifactId>
         <extensions>true</extensions>
         <configuration>
-          <javahOS>mac</javahOS>
-          <jdkIncludePath>/System/Library/Frameworks/JavaVM.framework/Headers</jdkIncludePath>
+          <javahOS>darwin</javahOS>
+          <jdkIncludePath>${java.home}/include</jdkIncludePath>
           <compilerStartOptions>
             <compilerStartOption>-fPIC -O</compilerStartOption>
           </compilerStartOptions>
```